internal/action/rcs.go

```diff
@@ -136,10 +137,10 @@ func (s *Action) RCSPush(c *cli.Context) error {
 	origin := c.Args().Get(0)
 	branch := c.Args().Get(1)
 
-	if origin == "" || branch == "" {
-		return ExitError(ExitUsage, nil, "Usage: %s git push <ORIGIN> <BRANCH>", s.Name)
-	}
 	if err := s.Store.RCSPush(ctx, store, origin, branch); err != nil {
+		if errors.Is(err, si.ErrGitNoRemote) {
+			return nil
```

Comment:
We might want to log that out occurred, no?

Reply:
Why bother if no remote is configured?
But yes, we could log this.